src/doc/gui_manual/WorkingWithFiles/Supported_File_Types.rst

```diff
@@ -3,32 +3,183 @@
 Supported File Types
 --------------------
 
-VisIt can create visualizations from databases that are stored in many types 
-of underlying file formats. VisIt has a database reader for each supported 
+VisIt_ can create visualizations from databases that are stored in many types 
+of underlying file formats. VisIt_ has a database reader for each supported 
 file format and the database reader is a plugin that reads the data from the 
-input file and imports it into VisIt. If your data format is not listed in 
+input file and imports it into VisIt._ If your data format is not listed in 
```

Comment:
Should the underscore come before the period?

Reply:
Yes. Thanks. Fixed now.